src/stories/components/ArrowPopover/ArrowPopover.tsx

```diff
@@ -0,0 +1,15 @@
+import styled from '@emotion/styled';
+import React from 'react';
+
+import type { PropsWithChildren } from 'react';
+type Props = PropsWithChildren;
```

Comment:
If the props are not going to add any other prop, just use `PropsWithChildren` directly

Reply:
for now just children